### PR TITLE
UXA for review confirm transfer flows

### PR DESCRIPTION
**Pull request by owillborn**
*Issue #:* /bcgov/entity#15895

*Description of changes:*
- Minor alignment and spacing fixes for Review and Confirm screen in Transfer flows

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]

File: ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <template>
   <v-card flat rounded id="submitting-party" class="mt-8 pa-8 pr-6">
     <v-row no-gutters>
-      <v-col cols="12" sm="2">
+      <v-col :cols="isMhrTransfer ? '3' : '12'"  :sm="isMhrTransfer ? '0' : '2'">
         <label class="generic-label" :class="{'error-text': validate}">Add Submitting Party</label>
       </v-col>
-      <v-col cols="12" sm="10" class="px-1">
+      <v-col :cols="isMhrTransfer ? '9' : '12'"  :sm="isMhrTransfer ? '0' : '10'" class="px-1">
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue]: Darn, this is a bit annoying..

**Review by cameron-eyds — APPROVED** [review]
Looks good to me! 
Fiddly 😵

File: ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue (hunk 1 above)
Comment (owillborn) [reply ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue]: fiddly design bits but not too bad

Comment (dimak1) [reply ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue]: Is there a way to keep it the same? I saw the comment from the designers, i think it should be consolidated at the design level.

Comment (dimak1) [reply ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue]: We cannot be this granular, it might break styling elsewhere....

Comment (dimak1) [reply ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue]: @owillborn maybe you could check with designers about this one?

Comment (owillborn) [reply ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue]: I'll chat with sara @dimak1

Comment (dimak1) [reply ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue]: Maybe show both pages with different columns and see which one it should be.

Comment (owillborn) [reply ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue]: @dimak1 changes made

Comment (dimak1) [reply ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue]: @owillborn thanks, what was the outcome? I see there are no changes in this file.

Comment (owillborn) [reply ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue]: @dimak1 Sorry for the late reply, talked with Sara and opted to remove the granular functionality in submitting party for this ticket, and create a separate ticket/rethink the design

Comment (dimak1) [reply ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue]: Perfect

**Review by dimak1 — APPROVED** [review]

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1285-scudo8tt.web.app

**Pull request merged**